docs/classical_control.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>Nit: "A <code>KeyCondition</code> will only trigger<strong>, and the operation will only be applied,</strong> if a preceding measurement..." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done.